getting-started/setup-multi-node-advanced.md

```diff
@@ -0,0 +1,370 @@
+access node needs to be configured for password authentication and
+needs the passwords for the relevant user on each data node.
+
+##### 5. Configure password encryption method
+This step is identical to (1) except applied to the access node.  Find the `postgresql.conf` file and set the `password_encryption` parameter:
+```
+password_encryption = 'scram-sha-256'
+```
+
+##### 6. Create a password file
+The password file enables the access node to connect securely to data 
+nodes and is by default located in the data directory ([PostgreSQL documentation][passfile]). If a file doesn't exist, create one with the 
+new user information.  Ex: from the terminal:
+
+```bash
+cd <datadir>
+cat >passfile <<EOT
+*:*:*:new_user:xyzzy
+EOT
+```
+Then apply the correct permissions to the file:
+```
+cd <datadir>
+chmod 0600 passfile
+```
+
+##### 7. Reload server configuration
+Reload the server configuration with:
+```
+pg_ctl reload
+```
+and the system setup is complete
+
+---
+
+## Certificate authentication [](multi-node-auth-certificate)
+This method is more complex to set up than password authentication, but 
+more secure and easier to automate.
+
+The steps to set up certificate authentication are:
+
+1. Set up a certificate authority that can sign certificates and
+   create a root certificate.
+2. Generate a key and certificate for the access node and data nodes.
+3. Configure the access node and data node to use the certificates and
+   to accept certified connections.
+4. Generate a key and certificate for each user.
+
+To use certificates, each instance involved in certificate
+authentication uses three files:
+
+- A *root certificate*, which we assumed to be named `root.crt`. This is 
+  the certificate that identifies a trusted Certification Authority (CA). 
+  It contains the public key of the CA (or CAs, in the event that there 
+  is a certificiation authority chain) and can be freely copied between 
+  servers.
+- A *certificate* that is signed by the CA. This contains the public
+  key and is used to verify signatures and can also be freely copied.
+- A *key* that contains the private key. This is used to sign messages
+  and should be kept secure on the instance where it is generated.
+
+For the access node a single *root certificate* can be shared among 
+users. However each user on the access node needs a unique *signed 
+certificate* and *key*.
+
+For each data node, the *root certificate* can be copied from the access 
+node, but a *signed certificate* and *key* must be generated.
+
+##### 1. Set up a Certification Authority
+
+A Certificate Authority (CA) is necessary as a trusted third party to 
+sign other certificates.  The _key_ of the CA is used to sign 
+Certificate Signing Requests (CSRs), and the _certificate_ of the CA is 
+what is used as a root certificate for other parties. Creating a new CA 
+is not necessary if there is already one available to be employed.  In 
+that case skip to the next step.
```

Comment:
```suggestion
The Certificate Authority (CA) provides the root of trust in the system and is used to create (sign) certificates for each TimescaleDB instance so that they can securely authenticate that they are communicating with trusted peers. This section describes how to create a self-signed root CA, but if a CA is already available to be employed, it is possible to skip to the next step.
```